lib/consumers.js

```diff
@@ -102,8 +102,8 @@ ConsumerInstance.prototype.__defineGetter__("uri", function() {
     return this.raw.base_uri;
 });
 
-ConsumerInstance.prototype.subscribe = function(topic) {
-    var stream = new ConsumerStream(this, topic);
+ConsumerInstance.prototype.subscribe = function(topic, requestDelay) {
```

Comment:
Should we use an options object instead of adding parameters here, and in the ConsumerStream constructor? The parameters before this patch are absolutely required, so having dedicated arguments for them makes sense. Here I'm worried that if we end up with other options subscribe() calls could get pretty messy.